lib/usdUfe/ufe/trf/UsdTransform3dUndoableCommands.cpp

```diff
@@ -16,43 +16,24 @@
 
 #include "UsdTransform3dUndoableCommands.h"
 
-#include <maya/MMatrix.h>
-#include <maya/MTransformationMatrix.h>
-#include <maya/MVector.h>
+#include <usdUfe/ufe/Utils.h>
+
 #include <ufe/transform3d.h>
 
-namespace MAYAUSD_NS_DEF {
-namespace ufe {
+namespace USDUFE_NS_DEF {
 
-MAYAUSD_VERIFY_CLASS_SETUP(Ufe::SetMatrix4dUndoableCommand, UsdSetMatrix4dUndoableCommand);
+USDUFE_VERIFY_CLASS_SETUP(Ufe::SetMatrix4dUndoableCommand, UsdSetMatrix4dUndoableCommand);
 
 UsdSetMatrix4dUndoableCommand::UsdSetMatrix4dUndoableCommand(
     const Ufe::Path&     path,
     const Ufe::Matrix4d& newM)
     : UsdUndoableCommand<Ufe::SetMatrix4dUndoableCommand>(path)
 {
-    // Decompose new matrix to extract TRS.  Neither GfMatrix4d::Factor
-    // nor GfTransform decomposition provide results that match Maya,
-    // so use MTransformationMatrix.
-    MMatrix m;
-    std::memcpy(m[0], &newM.matrix[0][0], sizeof(double) * 16);
-    MTransformationMatrix                xformM(m);
-    auto                                 t = xformM.getTranslation(MSpace::kTransform);
-    double                               r[3];
-    double                               s[3];
-    MTransformationMatrix::RotationOrder rotOrder;
-    xformM.getRotation(r, rotOrder);
-    xformM.getScale(s, MSpace::kTransform);
-    constexpr double radToDeg = 57.295779506;
-
-    _newT = Ufe::Vector3d(t[0], t[1], t[2]);
-    _newR = Ufe::Vector3d(r[0] * radToDeg, r[1] * radToDeg, r[2] * radToDeg);
-    _newS = Ufe::Vector3d(s[0], s[1], s[2]);
```

Comment:
This code was identical to the code moved into new file XformOpUtils.cpp. The only difference is this one set the values directly into member variables and the other was a function with three output vars.